Frameworks/CoreFoundation/Base.subproj/CFRuntime.c

```diff
@@ -275,6 +275,10 @@ void _CFRuntimeBridgeTypeToClass(CFTypeID cf_typeID, const void *cls_ref) {
     __CFUnlock(&__CFBigRuntimeFunnel);
 }
 
+uintptr_t __CFISAForTypeID(CFTypeID typeID) {
```

Comment:
Discussed offline, decided that this optimization wasn't worth the effort/complexity.

---
In reply to: [79034493](https://github.com/Microsoft/WinObjC/pull/1008#discussion_r79034493) [](ancestors = 79034493,79034377,79033450,79025442,79022634,78866010,78862337)